packages/admin-api-schema/lib/schemas/images.json

```diff
@@ -16,6 +16,11 @@
         "ref": {
           "type": ["string", "null"],
           "maxLength": 2000
+        },
+        "edited": {
+          "type": "string",
```

Comment:
We are using type `string` instead of `boolean` due to how the data is transmitted to the API via the image uploader in Ghost admin ([formData.append](https://developer.mozilla.org/en-US/docs/Web/API/FormData/append) is used which enforces the value to be a string)